bitrise.yml

```diff
@@ -120,6 +124,10 @@ workflows:
         - public_ids: "bcq-y54-rq7"
         - mobile_application_version: $DATADOG_UPLOADED_APPLICATION_VERSION_ID
         - fail_on_critical_errors: true
+    meta:
+      bitrise.io:
+        stack: osx-xcode-edge
+        machine_type_id: g2-m1.4core
```

Comment:
Can you move this up, just before `steps`? It's the most important part of `test-on-Xcode-stack`

Reply:
and same for this other PR